dashboard/app/controllers/application_controller.rb

```diff
@@ -167,6 +167,13 @@ def milestone_response(options)
 
     # logged in users can:
     if current_user
+      # participate in A/B testing
+      # as of March 25, 2016, the text of the block hint button is being
+      # A/B tested. See @cdo/apps/src/templates/DialogButtons.jsx.
+      if Gatekeeper.allows('ab_testing', default: true)
```

Comment:
Consider naming the Gatekeeper flag something specific to your use case, e.g., ab_testing_hint_text.